src/common/Navigation/index.js

```diff
@@ -71,7 +74,7 @@ const Navigation = () => {
               handleOpen={() => setOpen(true)}
             />
           </BurgerItem>
-          {open &&
+          {open && (
```

Comment:
Mógłbyś wyjaśnić do czego służy ten open ? :)

Reply:
To akurat Kod Wojtka, u mnie go tylko sformatowało, ale generalnie jak atrybut open jest przekazany do komponentu to  renderuje się to co jest w tym fragmencie po prawej od niego. Jeżeli open nie jest przekazane to to się nie wyświetla. Innymi słowy to mechanizm otwierania menu